extensions/binding/org.eclipse.smarthome.binding.fsinternetradio/src/main/java/org/eclipse/smarthome/binding/fsinternetradio/internal/FSInternetRadioDiscoveryParticipant.java

```diff
@@ -91,17 +102,31 @@
         // as reported in: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/7
         final Set<String> revoRadios = new HashSet<String>();
         SUPPORTED_RADIO_MODELS.put("REVO TECHNOLOGIES LTD", revoRadios);
-        revoRadios.add("SUPERCONNECT");
+        revoRadios.add("S10");
 
         // as reported in: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/10
+        // and: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/21
         final Set<String> robertsRadios = new HashSet<String>();
         SUPPORTED_RADIO_MODELS.put("ROBERTS RADIO LIMITED", robertsRadios);
         robertsRadios.add("ROBERTS STREAM 93I");
+        robertsRadios.add("ROBERTS STREAM 83I");
 
         // as reported in: https://community.openhab.org/t/internet-radio-i-need-your-help/2131/11
-        final Set<String> AUNARadios = new HashSet<String>();
-        SUPPORTED_RADIO_MODELS.put("AUNA", AUNARadios);
-        AUNARadios.add("CONNECT 150");
```

Comment:
Did this id change? Do you have a reference?

Reply:
same as above.